src/pages/CreateEvent/ChallengeDetails.scss

```diff
@@ -39,30 +39,28 @@
       .date-input-container {
         display: flex;
         align-items: center;
-        border: 2px solid #000;
+        border: 3px solid #000;
+        flex-direction: row;
         border-radius: 6px;
         padding: 0.5rem;
         font-size: 1rem;
         gap: 0.5rem;
-        
-        .icon {
-          width: 32px;
-        }
-    
+        width: 80%;
+  
         input {
-          border: none;
           background: transparent;
           outline: none;
           font-family: inherit;
           font-size: inherit;
           cursor: pointer;
+          border: none;
         }
-    
+  
         .divider {
           color: #000;
           font-size: 1rem;
         }
-    
+  
         &:hover {
           border-color: #0056b3;
         }
```

Comment:
change color to a variable

Reply:
Do you mean Tailwind or an embedded style?

Reply:
Using tailwind you can also use variable colour codes, it can be found in the tailwind.config.js